examples/artemis-basic-deployment.yaml

```diff
@@ -2,7 +2,11 @@ apiVersion: broker.amq.io/v1beta1
 kind: ActiveMQArtemis
 metadata:
   name: ex-aao
+  namespace: activemq-artemis-operator
 spec:
   deploymentPlan:
     size: 1
     image: placeholder
+  brokerProperties: {
+    globalMaxSize: 512MB,
```

Comment:
we would need to do some extra work to support some sort of postfix or type modifier  - MB,GB etc.
probably best if the broker extracts those. I think that will be preferable to having just a raw number there.
the operator could also support these but I think it is broker config, rather than kube resource references etc.

Reply:
support in https://github.com/apache/activemq-artemis/pull/3959